maquetado/scss/details.scss

```diff
@@ -0,0 +1,25 @@
+.book-details {
+  p {
+    display: block;
+    padding: 5px;
+  }
+  span {
+    color: grey;
```

Comment:
Create `colors` a file and add variables with the colors in hexa

Reply:
Done